domains/thebestdev.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "notthebestdev",
+        "email": "thebestdeveloper@outlook.fr"
+    },
+    "record": {
+        "TXT": "dh=d04b3166526bba7742cb84fb32a18b421b85325a"
```

Comment:
then why is this .json file doing here?

Reply:
idk I made it with the dashboard

Reply:
you'd have to delete it in your fork to remove it in this PR btw